### DISCUSSION TITLE
build fail on windows with "./scripts/build.ps1 -Tls openssl" , have anybody ever encountered this error? Please Help

### DISCUSSION THREAD

**Discussion by Ang0426** [microsoft/msquic#3472] [Q&A]
With Visual Studio 2019, and with "-Tls schannel", it can success. 

The error is :

C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(241,5): error MSB8066: “D:\msquic_2.1.7\msquic-2.1.7\msquic-2.1.7\build\windows\x64_openssl\CMakeFiles\1f074b3e745d13
ad1e50bbd0d94cd13a\ae264b26276f733d.rule;D:\msquic_2.1.7\msquic-2.1.7\msquic-2.1.7\build\windows\x64_openssl\CMakeFiles\1f074b3e745d13ad1e50bbd0d94cd13a\fb1ce47af5ee18db.rule;D:\msquic_2.1.7\msquic-2.1.7\msquic-2.1.7\build\windows\x64_openssl\CMakeFiles\1f074b3e745d13ad1e50bbd0d94cd13a\OpenSSL_Target.rule”exited with code 9009。 [D:\msquic_2.1.7\msquic-2.1.7\msquic-2.1.7\build\windows\x64_openssl\_deps\opensslquic-build\OpenSSL_Target.vcxproj]

**Comment by nibanks** [comment]
No, I haven't seen this, nor do I know what to do for it.
